product_contract_transmit_method/README.rst

```diff
@@ -0,0 +1,73 @@
+================================
+Product Contract Transmit Method
```

Comment:
I think the name is misleading, as it is a sale order to contract link, not dealing with product. The `product_contract` family deals precisely with products linked to contracts. I suggest `sale_contract_transmit_method` or `contract_sale_transmit_method` to align it to the naming of other similar modules.

Reply:
+1